config/manager/kustomization.yaml

```diff
@@ -1,2 +1,7 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
```

Comment:
Please run `make reset` to get rid of this.

Reply:
done. unnecessary change removed.